services/dis-apim-operator/api/v1alpha1/apiversion_types.go

```diff
@@ -208,6 +248,10 @@ func (av *ApiVersion) GetApiVersionAzureFullName() string {
 	return fmt.Sprintf("%s-%s", av.Namespace, av.Name)
 }
 
+func (av *ApiVersion) GetApiVersionDiagnosticAzureFullName() string {
+	return "applicationinsights"
```

Comment:
nitpick: I think you mentioned you had some troubles figuring out what this was.
A comment here would be helpful for posteriority.

Reply:
Related with [this](https://github.com/Altinn/altinn-platform/blob/f90e4b4df1ff758fdc1777b05f43c07998a93a8f/services/dis-apim-operator/internal/azure/azureclient_enums.go#L5-L10)?

Reply:
Yeah, that function is not in use so I will remove it. I can document the troubles in the type you linked to